FitNesseRoot/DbFit/AcceptanceTests/CoreTests/StoredProcNoParams/content.txt

```diff
@@ -1,5 +1,9 @@
 !3 If there are no parameters, Execute Procedure needs just one row
 
+|Clean|
+|table|clean?|
+|users|true|
```

Comment:
The `Users` table is persistant. DML usually gets rolled back, automatically in flow mode, and explicitly via the `Rollback` in the stand alone mode tests.

The Informix tests, running in Informix mode, auto commit so the data hangs around and causes a problem for the next time the test is run (in either of the modes).